bundle-size/app.js

```diff
@@ -562,51 +562,39 @@ module.exports = app => {
 
   v0.post('/commit/:headSha/store', async (request, response) => {
     const {headSha} = request.params;
-    const {gzippedBundleSize, brotliBundleSize} = request.body;
+    const {brotliBundleSize} = request.body;
 
     if (request.body['token'] !== process.env.TRAVIS_PUSH_BUILD_TOKEN) {
       return response.status(403).end('You are not Travis!');
     }
-    for (const bundleSize of [gzippedBundleSize, brotliBundleSize]) {
-      if (typeof bundleSize !== 'number') {
-        return response
-          .status(400)
-          .end(
-            'POST request to /store must have numeric fields ' +
-              '"gzippedBundleSize" and "brotliBundleSize"'
-          );
-      }
+    if (typeof brotliBundleSize !== 'number') {
+      return response
+        .status(400)
+        .end(
+          'POST request to /store must have numeric field "brotliBundleSize"'
+        );
     }
 
-    // TODO(danielrozenberg): remove this block once we switch entirely to the
+    // TODO(danielrozenberg): remove this section once we switch entirely to the
     // JSON format.
-    for (const [compression, extension, bundleSizeValue] of [
-      ['gzip', '', gzippedBundleSize],
-      ['brotli', '.br', brotliBundleSize],
-    ]) {
-      const bundleSizeText = `${bundleSizeValue}KB`;
-      const bundleSizeFile = `${headSha}${extension}`;
-
-      try {
-        await getBuildArtifactsFile(userBasedGithub, bundleSizeFile);
-        app.log(
-          `The file bundle-size/${bundleSizeFile} already exists in the ` +
-            'build artifacts repository on GitHub. Skipping...'
-        );
-        continue;
-      } catch (unusedException) {
-        // The file was not found in the GitHub repository, so continue to
-        // create it...
-      }
+    const bundleSizeText = `${brotliBundleSize}KB`;
+    const bundleSizeFile = `${headSha}.br`;
 
+    try {
+      await getBuildArtifactsFile(userBasedGithub, bundleSizeFile);
+      app.log(
+        `The file bundle-size/${bundleSizeFile} already exists in the ` +
+          'build artifacts repository on GitHub. Skipping...'
+      );
+    } catch (unusedException) {
```

Comment:
It looks like this try block was moved into the `catch`; is that intentional? The comment that was removed seemed useful.

Reply:
Yes, it's intentional - before that there was a `continue;` line above for when this needed to be skipped, but now since there's no for loop here, I had to move this block inside. I'll restore the comment...